gaphor/diagram/general/metadata.py

```diff
@@ -0,0 +1,206 @@
+from gaphor.core.modeling.properties import attribute
+from gaphor.core.styling import (
+    Style,
+    TextAlign,
+    VerticalAlign,
+    FontWeight,
+    JustifyContent,
+)
+from gaphor.diagram.presentation import ElementPresentation
+from gaphor.diagram.shapes import (
+    Box,
+    Text,
+    draw_border,
+    draw_left_separator,
+    draw_top_separator,
+    Orientation,
+)
+from gaphor.i18n import gettext
+
+
+class MetadataItem(ElementPresentation):
+    createdBy: attribute[str] = attribute("createdBy", str, "")
+    description: attribute[str] = attribute("description", str, "")
+    website: attribute[str] = attribute("website", str, "")
+    revision: attribute[str] = attribute("revision", str, "")
+    license: attribute[str] = attribute("license", str, "")
+    createdAt: attribute[str] = attribute("createdAt", str, "")
+    updatedAt: attribute[str] = attribute("updatedAt", str, "")
```

Comment:
I would expect that Created/Updated At would be for a location, I suggest Created On and Updated On would be more clear for a date.

Reply:
fixed